### PR TITLE
CDS-1679 place filters in the same line

### PR DESCRIPTION
**Pull request by bkolada**
- [x] place all elements in one line in different cols, buttons needs work
- [x] simplify css top margin(i think it's overcomplicated) [done for tags, results, and strange box on the right]
- [x] correct buttons for unstable and branches to show proper graphic on it 
- [x] check what depends from classes you changed 
- [x] check how it looks like to have different browser size
- [x] check functionality
- [ ] AFTER REVIEW create new branch and cherry pick commits

### CODE REVIEW

**Comment by campos-ddc** [comment]
Could you remake this PR targeting the `staging` branch instead of `katana`?

**Comment by bkolada** [comment]
@campos-ddc so now it's against staging.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+EXPOSE 8001
+EXPOSE 8010
+
+ADD start-autobahn.sh /start-autobahn.sh
+RUN chmod +x /start-autobahn.sh
+
+VOLUME /katana
+VOLUME /buildmaster
+VOLUME /buildmaster_configuration
```
Comment (campos-ddc) [inline Dockerfile:23]: As @mariangemarcano mentioned in another PR, we dno't need `buildmaster_configuration` here, just `buildmaster` should be enough for us to create a test server

File: master/requirements-dev.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Jinja2 == 2.8
+sqlalchemy >= 0.6, <= 0.7.9
+sqlalchemy-migrate == 0.7.2
+python-dateutil == 1.5
+autobahn == 0.11.0
+python-ldap
+mysql-python == 1.2.5
+pymysql == 0.7.1
+PyJWT == 1.4.0
+psutil == 4.3.0
```
Comment (campos-ddc) [inline master/requirements-dev.txt:11]: Aren't any of these declared in `setup.py`? IMO `requirements-dev` should only contain things that are required for development but not every day use (the every day stuff should be in setup.py)

**Comment by bkolada** [comment]
@campos-ddc These changes you commented so far are from another pr/branch which I used to develop that's why they are visible here. They will not be merged nor should be considered as a part of current pr. After accepted review I will create new branch with cherry picked changes from current and will merge that. I should mention that before, sorry.

**Comment by bkolada** [comment]
I closed that pr as I think it will be much better just to create new one and have exact differences.

**Pull request closed without merging**